src/charclass.cpp

```diff
@@ -27,19 +27,35 @@
 
 void initClass(int player)
 {
+    // TODO: Dynamic class list loading/saving from/to file
+    enum class classes_t
```

Comment:
It does require the extra typing of PlayerClass::BARBARIAN, however I added the "class" modifier to prevent scoping issues. The casting is an issue, and I believe I made a few TODOR:'s to look into dealing with a few of those.

There's many instances of ints being used as bools, and eliminating them is something I'd like to continue to look into. Creating an enum was one such method, but only the first step.